pkg/controller/bypass/bypass_controller.go

```diff
@@ -0,0 +1,268 @@
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"kmesh.net/kmesh/pkg/logger"
+	"kmesh.net/kmesh/pkg/utils"
+)
+
+var (
+	log = logger.NewLoggerField("bypass")
+	FS  embed.FS
+)
+
+const (
+	DefaultInformerSyncPeriod = 30 * time.Second
+	LabelSelectorBypass       = "kmesh.net/bypass=enabled"
+)
+
+func StartByPassController(client kubernetes.Interface) error {
+	stopChan := make(chan struct{})
+	nodeName := os.Getenv("NODE_NAME")
+
+	informerFactory := informers.NewSharedInformerFactoryWithOptions(client, DefaultInformerSyncPeriod,
+		informers.WithTweakListOptions(func(options *metav1.ListOptions) {
+			options.FieldSelector = fmt.Sprintf("spec.nodeName=%s", nodeName)
+			options.LabelSelector = LabelSelectorBypass
+		}))
+
+	informerFactory.Start(wait.NeverStop)
+	informerFactory.WaitForCacheSync(wait.NeverStop)
+
+	podInformer := informerFactory.Core().V1().Pods().Informer()
+
+	podInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			pod, ok := obj.(*corev1.Pod)
+			if !ok {
+				log.Errorf("expected *corev1.Pod but got %T", obj)
+				return
+			}
+
+			log.Infof("%s/%s: ADDED", pod.GetNamespace(), pod.GetName())
+			enableSidecar, _ := checkSidecar(client, pod)
+			if !enableSidecar {
+				log.Info("do not need add iptables rules, pod is not managed by sidecar")
+				return
+			}
+
+			nspath, _ := getnspath(pod)
+			addIptables(nspath)
+		},
+		DeleteFunc: func(obj interface{}) {
+			if _, ok := obj.(cache.DeletedFinalStateUnknown); ok {
+				return
+			}
+			pod, ok := obj.(*corev1.Pod)
+			if !ok {
+				log.Errorf("expected *corev1.Pod but got %T", obj)
+				return
+			}
+
+			if isPodBeingDeleted(pod) {
+				log.Debugf("%s/%s: Pod is being deleted, skipping further processing", pod.GetNamespace(), pod.GetName())
+				return
+			}
+			enableSidecar, _ := checkSidecar(client, pod)
+			if !enableSidecar {
+				log.Info("do not need delete iptables rules, pod is not managed by sidecar")
+				return
+			}
+			log.Infof("%s/%s: DELETED", pod.GetNamespace(), pod.GetName())
+			nspath, _ := getnspath(pod)
+			deleteIptables(nspath)
+		},
+	})
+
+	go podInformer.Run(stopChan)
+
+	return nil
+}
+
+func isPodBeingDeleted(pod *corev1.Pod) bool {
+	return pod.ObjectMeta.DeletionTimestamp != nil
+}
+
+func addIptables(ns string) error {
+	iptArgs := [][]string{
+		{"-t", "nat", "-I", "PREROUTING", "1", "-j", "RETURN"},
+		{"-t", "nat", "-I", "OUTPUT", "1", "-j", "RETURN"},
+	}
+
+	execFunc := func(netns.NetNS) error {
+		log.Infof("Running add iptables rule in namespace:%s", ns)
+		for _, args := range iptArgs {
+			if err := utils.Execute("iptables", args); err != nil {
+				return fmt.Errorf("failed to exec command: iptables %v\", err: %v", args, err)
+			}
+		}
+		return nil
+	}
+	if err := netns.WithNetNSPath(ns, execFunc); err != nil {
+		return fmt.Errorf("enter ns path: %v, run command failed: %v", ns, err)
```

Comment:
```suggestion
		return fmt.Errorf("enter namespace path: %v, run command failed: %v", ns, err)
```
In error messages, try not to apply abbreviations.

Reply:
> In error messages, try not to apply abbreviations.

done